### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by glitchform**
Made some minor changes to some parts of the FAQ.

### CODE REVIEW

File: docs/en/faq.md [hunk 1]
```diff
@@ -2,44 +2,44 @@
...
 
 ## I know LaTeX and want to use it inside my Markdown files as well. Is this possible?
 
-Yes. Simply write your `LaTeX`-statements where you want them. As soon as you export to PDF, Pandoc will take care the statements will be interpreted by the PDF engine. Unfortunately, `LaTeX` syntax highlighting is not supported. Also, please note that Pandoc will clear all `LaTeX`-blocks prior to exporting to anything other than PDF, which means that blocks within `\begin` and `\end`, for instance, will be missing completely from the final Office file. On HTML-export, all `LaTeX`-blocks will retained, but not converted to something else.
+Yes. Simply write your `LaTeX`-statements where you want them. As soon as you export to PDF, Pandoc will take care of the rest and the statements will be interpreted by the PDF engine. Unfortunately, `LaTeX` syntax highlighting is not supported. Also, please note that Pandoc will clear all `LaTeX`-blocks prior to exporting to anything other than PDF, which means that blocks within `\begin` and `\end`, for instance, will be missing completely from the final Office file. On HTML-export, all `LaTeX`-blocks will retained, but not converted to something else.
```
Comment (nathanlesage) [inline docs/en/faq.md]: Just noticed while reading — the last sentence is not correct English, right? "all LaTeX-blocks will **be** retained …" is correct, or? Just wondering …

**Comment by nathanlesage** [comment]
Awesome, thank you! :)

Concerning that one comment, I think I missed the "be", didn't I?

**Comment by glitchform** [comment]
Yes, you are right, "be" is missing

**Comment by nathanlesage** [comment]
Alright, I'll add it quickly! Thanks :)

**Comment by glitchform** [comment]
Cool!

**Pull request merged**